app/main.py

```diff
@@ -1 +1,42 @@
-#  write your code here
+from typing import List
+
+
+class Animal:
+    def __init__(
+            self, name: str, appetite: int, is_hungry: bool = True
```

Comment:
Place each argument including `self` on a new line when they include type annotation and there is a need to make line breaking. It will help to improve the code style and readability